firstapp/app/src/androidTest/java/com/example/firstapp/ShellUtility.java

```diff
@@ -488,7 +490,7 @@ public void logData(long[][] allActionStamps) {
      * Executes a CUJ with preparatory actions pre and measured actions post.
      * Caches data and then runs through the CUJ iterations times and logs results
      ******************************************************************************/
-    public void executeCUJ(String[] preCUJ, String[] postCUJ, int iterations) throws Exception {
+    public void executeCUJ(String[] preCUJ, String[] postCUJ, int iterations, long recordingBuf) throws Exception {
```

Comment:
What is recordingBuf?  Why is it needed?  Needs a more descriptive name.  And if it is time-based, then should append time unit to the end of the name.

Reply:
Time we leave at the beginning and end of measured section so we can get a clip in which the CUJ in question is clear